process_subways.py

```diff
@@ -94,8 +94,16 @@ def calculate_way_center(el):
             return
         center = [0, 0]
         count = 0
-        for nd in el['nodes']:
+        way_nodes = el['nodes']
+        for i, nd in enumerate(way_nodes):
             if nd in nodes:
+                # Don't count the first node of a closed way twice
+                if (
+                        len(way_nodes) > 1
+                        and i == len(way_nodes) - 1
```

Comment:
NIT: put `len(way_nodes` as a variable earlier. And check, whether i is the last index at the first place, then check other conditions.

Reply:
\+